go.mod

```diff
@@ -8,9 +8,11 @@ require (
 	code.gitea.io/gitea v1.22.1
 	code.gitea.io/sdk/gitea v0.19.0
 	github.com/AlecAivazis/survey/v2 v2.3.7
+	github.com/antihax/optional v1.0.0
```

Comment:
What is this library is good for ? (we need to be careful which lib we want to add to the project, everything has a cost)

Reply:
@chmouel it's underline dependency for `gdasson/bitbucketv1go` ([used here in my PR](https://github.com/zakisk/pipelines-as-code/blob/f515e53edd51f723cfb8324eb6b6e939c953e1cf/test/pkg/bitbucketserver/pr.go#L35))